src/Blt/Plugin/Commands/SwsCommands.php

```diff
@@ -366,4 +367,103 @@ protected function getSiteAliases() {
     return $this->siteAliases;
   }
 
+  /**
+   * After code deployed, update all sites on the stack.
+   *
+   * @command sws:post-code-deploy
+   *
+   * @aliases sws:post-code-update
+   */
+  public function postCodeDeployUpdate($target_env, $deployed_tag, $options = ['no-slack' => FALSE]) {
+    $sites = $this->getConfigValue('multisites');
+    $parallel_executions = (int) getenv('UPDATE_PARALLEL_PROCESSES') ?: 10;
```

Comment:
Can we add to the annotation that this ENV variable needs to be set?

Reply:
Well, not that it _needs_ to be, but that it can be.

Reply:
it doesn't need to be set. it will default to 10 parallel processes. But we can definitely add the information to the docblock

Reply:
Cool. now `blt sws:post-code-deploy --help` will show that help information.